docs/using.rst

```diff
@@ -804,6 +815,12 @@ By default, the following locations are searched:
   ``~/.config/letsencrypt/cli.ini`` if ``$XDG_CONFIG_HOME`` is not
   set).
 
+Since this configuration file applies to all invocations of certbot it is incorrect
+to list domains in it. Listing domains in cli.ini will prevent renewal from working.
```

Comment:
Another nit:

> Listing domains in cli.ini will prevent renewal from working.

This is no longer true since 0.14.0 and #4479. This is a hack though and has caused problems like #4774 but technically we "properly" handle this case now.

Up to you what you'd like to do with this information.